panel/models/echarts.py

```diff
@@ -62,4 +62,4 @@ def __js_skip__(cls):
 
     renderer = Enum("canvas", "svg")
 
-    theme = Enum("default", "light", "dark")
+    theme = String("default")
```

Comment:
This will enable users to add the ECharts theme `.js` files manually as in the `echarts.md` example.